### DISCUSSION TITLE
Config value 'plugins': The "social" plugin is not installed

### DISCUSSION THREAD

**Discussion by pavelklymenko** [squidfunk/mkdocs-material#5639] [Q&A] [answered]
Tried to install insiders following the guidelines at [Getting started with Insiders -> Installation](https://squidfunk.github.io/mkdocs-material/insiders/getting-started/)

**Environment**

```
Mac OS 13.4.1 (22F82)/ Apple M1 Max
Python 3.11.4
```

**Here's the procedure I followed:**

```
python3 -m venv .insiders
source .insiders/bin/activate
pip install git+https://${GH_TOKEN}@github.com/squidfunk/mkdocs-material-insiders.git
```

**Configuration**
_mkdocs.yml_
```
site_name: Blog
```

_mkdocs.insiders.yml_
```
INHERIT: mkdocs.yml
plugins:
  - search
  - social
  - tags
```
After running the command `mkdocs build --config-file mkdocs.insiders.yml` get the following error (for Social and Tags)

```
blog ❯ mkdocs build --config-file mkdocs.insiders.yml                                               Py .insiders 16:01:32
ERROR    -  Config value 'plugins': The "social" plugin is not installed
Aborted with 1 Configuration Errors!
```

Already tried to what was described in [#3713](https://github.com/squidfunk/mkdocs-material/discussions/3713) and [#3177](https://github.com/squidfunk/mkdocs-material/discussions/3177)

**Comment by kamilkrzyskow** [comment]
Your base MkDocs.yml file seems to be missing the 
```yml
theme:
    name: material
```
configuration part.
The social plugin is a part of the material theme and by default when no theme name is given the MkDocs theme is used by default which doesn't have any social plugin.

https://squidfunk.github.io/mkdocs-material/creating-your-site/#minimal-configuration

**Reply by pavelklymenko** [reply]
I tried to add that as well, still had the same error message. Thanks for suggestion, tho.

**Comment by pavelklymenko** [comment, marked as answer]
What I tried to do is move all the `plugin` configurations to the `mkdocs.yml` and run `mkdocs build --config-file mkdocs.yml`. This time it reported an error:
`ERROR    -  Required dependencies of "social" plugin not found. Install with: pip install pillow cairosvg`

Fixed that with `pip install pillow cairosvg`

**Reply by pavelklymenko** [reply]
And I guess this is what I didn't get without @kamilkrzyskow suggestion. Thanks!

**Reply by kamilkrzyskow** [reply]
This is a completely different error, which shouldn't even appear when running `mkdocs build --config-file mkdocs.yml`, because it doesn't have any plugins section to begin with, therefore the social plugin shouldn't report any issues.

My suggestion was the answer for the error in the OP post. You went ahead a few steps and solved your nth error and marked it as the answer to the OP, when actually it is a different issue 🙄. 

Nonetheless I'm glad you figured it out.
